Mathlib/Analysis/SpecialFunctions/Pow/Complex.lean

```diff
@@ -238,6 +238,9 @@ theorem conj_cpow (x : ℂ) (n : ℂ) (hx : x.arg ≠ π) : conj x ^ n = conj (x
 theorem cpow_conj (x : ℂ) (n : ℂ) (hx : x.arg ≠ π) : x ^ conj n = conj (conj x ^ n) := by
   rw [conj_cpow _ _ hx, conj_conj]
 
+lemma cpow_natCast_add_one_ne_zero (n : ℕ) (z : ℂ) : (n + 1 : ℂ) ^ z ≠ 0 :=
```

Comment:
This looks like `natCast_add_one_cpow_ne_zero`, since the `natCast` is left of the `^`.

Reply:
OK, will change.

Reply:
#20406